fdbserver/DeltaTree.h

```diff
@@ -391,7 +391,7 @@ struct DeltaTree {
 			DecodedNode* n = root;
 			bool addLeftChild = false;
 
-			while (n != nullptr) {
+			while (true) {
```

Comment:
Oops, I approved too soon.  This change also must be reverted.  If insert is attempted on an empty DeltaTree the assert after the loop will block it, this loop's conditional should still just be `n != nullptr`.

Reply:
Rethinking this, let's just add 
```
if(root == nullptr) {
  return false;
}
```
at the start and update the comment to say that insertion on an empty tree returns false as well.

Reply:
Then the while loop can be `while(true)` and the assertion is not needed as n can't be null at that point.